pipeline/filters/aws-metadata.md

```diff
@@ -49,9 +52,84 @@ $ bin/fluent-bit -c /PATH_TO_CONF_FILE/fluent-bit.conf
     account_id true
     hostname true
     vpc_id true
+    tags_enabled true
 
 [OUTPUT]
     Name stdout
     Match *
 ```
 
+## EC2 Tags
+
+EC2 Tags are a useful feature that enables you to label and organize your EC2 instances by creating custom-defined key-value pairs. These tags are commonly utilized for resource management, cost allocation, and automation. Consequently, including them in the Fluent Bit generated logs is almost essential.
+
+To achieve this, AWS Filter can be configured with `tags_enabled true` to enable the _tagging_ of logs with the relevant EC2 instance tags. This setup ensures that logs are appropriately tagged, making it easier to manage and analyze them based on specific criteria.
+
+### Requirements
+
+For Fluent Bit to retrieve EC2 tags, the [instance-metadata-tags](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ec2/modify-instance-metadata-options.html) option must be enabled on the EC2 instance where it is running. If this option is not enabled, Fluent Bit will not be able to fetch the tags, resulting in its inability to identify them. As a consequence, Fluent Bit will operate as if there are no tags associated with the EC2 instance.
+
+That's it! However, proper configuration is still necessary, and this will be elaborated on in the following section.
+
+### Configuration
```

Comment:
I think this is too repetitive, let's not have the config options here again, let's just note the users can read them in the main config section

Reply:
Sure, I deleted the 'Configuration' from the EC2 Tags section. You're right that our table with all config options should be sufficient.